it/controllers/StubControllerISpec.scala

```diff
@@ -0,0 +1,72 @@
+package controllers
+
+import itutil.IntegrationSpecBase
+import itutil.config.IntegrationTestConstants.{testJourneyId, testMinimalLevelJourneyConfigV2}
+import model.{JourneyConfigV2, JourneyDataV2, JourneyOptions}
+import play.api.http.HeaderNames
+import play.api.http.Status.OK
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.Json
+import play.api.{Application, Environment, Mode}
+import services.IdGenerationService
+
+class  StubControllerISpec extends IntegrationSpecBase {
+
+  object MockIdGenerationService extends IdGenerationService {
+    override def uuid: String = testJourneyId
+  }
+
+  override implicit lazy val app: Application = new GuiceApplicationBuilder()
+    .in(Environment.simple(mode = Mode.Dev))
+    .bindings(bind[IdGenerationService].toInstance(MockIdGenerationService))
+    .configure(fakeConfig("application.router" -> "testOnlyDoNotUseInAppConf.Routes"))
+    .build
+
+  s"${controllers.testonly.routes.StubController.showStubPageForJourneyInitV2().url}" should {
+    "return 200" in {
+      val res = buildClientTestOnlyRoutes(path = "v2/test-setup")
+        .withHeaders(HeaderNames.COOKIE -> sessionCookieWithCSRF, "Csrf-Token" -> "nocheck")
+        .get()
+      await(res).status shouldBe 200
```

Comment:
Easier to read/more idiomatic to use play status constants. Plus, that's what we've done in the rest of the tests now so would make sense to be consistent.

Reply:
I agree with this one